README.md

```diff
@@ -1,6 +1,10 @@
 # BSON parser
 
-If you don't yet know what BSON actually is, read [the spec](http://bsonspec.org).
+BSON is short for Bin­ary JSON and is the bin­ary-en­coded seri­al­iz­a­tion of JSON-like doc­u­ments. You can learn more about it in [the specification](http://bsonspec.org).
+
+This library is a native (C++) parser for Node.js that takes advantage of low-level bindings to be faster and use less memory than the default (but browser-friendly) JavaScript parser (located at [mongodb/js-bson](https://github.com/mongodb/js-bson)).
```

Comment:
I kind of think we should be careful with the term `native` here. Instead maybe we say:

```
This module is a C++ addon for Node.js that implements the
BSON parsing and serialization outside of the JavaScript environment
in order to increase speed and memory efficiency for certain workloads.
It uses the same types as the pure JavaScript implementation, which
can be found at [mongodb/js-bson](https://github.com/mongodb/js-bson)
```

something more like that